docs/reference/ccr/getting-started.asciidoc

```diff
@@ -306,36 +324,34 @@ POST /server-metrics-follower/_ccr/unfollow
 [[ccr-getting-started-auto-follow]]
 ==== Automatically create follower indices
```

Comment:
See comment above. I'm also wondering how this changes with Data streams?

Reply:
CCR doesn't support data streams currently. It looks like that's being added here: https://github.com/elastic/elasticsearch/pull/61993